codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/AsyncClientClass.java

```diff
@@ -282,4 +287,18 @@ private CodeBlock createEventStreamTaggedUnionJsonMarshaller(ShapeModel eventStr
     private TypeName eventStreamType(ShapeModel shapeModel) {
         return poetExtensions.getModelClass(shapeModel.getShapeName());
     }
+
+    private MethodSpec utilitiesMethod() {
+        UtilitiesMethod config = model.getCustomizationConfig().getUtilitiesMethod();
+        ClassName returnType = PoetUtils.classNameFromFqcn(config.getReturnType());
+
+        return MethodSpec.methodBuilder(config.getName())
```

Comment:
Does the name need to be configurable, or should it always be utilities() for consistency?

Reply:
Removed the "name" from configuration for now to maintain consistency.